maxim/tests/elevenlabs/stt_tts_pipeline_agent.py

```
@@ -0,0 +1,189 @@
+from maxim import Maxim
+from maxim.logger.components.trace import TraceConfigDict
+from maxim.logger.elevenlabs import instrument_elevenlabs_stt_tts
+
+load_dotenv()
+
+# Configuration
+ELEVENLABS_API_KEY = os.getenv("EL_API_KEY")
+MAXIM_BASE_URL = os.getenv("MAXIM_BASE_URL")
+
+if not ELEVENLABS_API_KEY:
+    raise ValueError("ELEVENLABS_API_KEY environment variable is not set")
```

Ruff (0.14.3)

Avoid specifying long messages outside the exception class

(TRY003)